python/ray/serve/tests/test_router.py

```diff
@@ -204,11 +212,11 @@ def get_queues(self):
     worker = MockWorker.remote()
     q = ray.remote(VisibleRouter).remote()
     await q.setup.remote()
-    BACKEND_NAME = "max-concurrent-test"
+    backend_name = "max-concurrent-test"
```

Comment:
somehow lint starts complaining about this casing issue.

Reply:
weird